pkg/translator/cloudwatch/metadata.yaml

```diff
@@ -0,0 +1,3 @@
+status:
+  codeowners:
+    active: []
```

Comment:
Please add yourself as a code owner here.

Reply:
It still fails since I am not a member. Any idea of the next step?

Reply:
It's not a required test, so I will let it fail.